strings/quest_workers.json

```
@@ -1,116 +1,116 @@
 {
 	"not_complete_phrases": [
-		"You couldnt do it? What the hell are you even doing there?",
-		"It's a shame, we could've helped each other",
-		"Why did you even take this on?",
-		"Our department will surely be underpaid for being behind schedule",
-		"Listen here, moron, if you mess with our plans again, I'll pay you a visit and disembowel your dog",
-		"Observing. The delivery request has been ignored by the station crew. The list of perpetrators was identified. The Department will be notified about this. ",
-		"We are sure as shit getting sacked this term. Hey, which one of you can I stay with for a couple of months?",
-		"Section 4 first, then revisors, and now you? I need to feed my family, but what do YOU know about such matters.",
-		"Hey, boys! Put yourselves together, our work is stalling because of you.",
-		"Good work. Mister S will appreciate your efforts.",
-		"I'LL MAKE SURE YOU BECOME UNEMPLOYED THE NEXT WEEK YOU BRAINDEAD LUMP OF MEAT",
-		"Cheers, dudes, I really didn't want to work today. ",
-		"Shit. It seems like we're not getting our delivery today. "
+		"Не вышло? Вы чем там занимаетесь вообще?",
+		"Жаль, мы могли бы помочь друг другу.",
+		"Зачем вы вообще брались за это?",
+		"Нашему отделу наверняка урежут зарплату за отставание в графике.",
+		"Слушай сюда, кретин, ещё раз вы будете срывать наши планы, я приду к тебе домой и убью твою собаку.",
+		"Наблюдение. Запрос на доставку проигнорирован экипажем станции. Круг лиц определён, Ведомство узнает об этом.",
+		"В этом квартале нас точно сократят. Эй, к кому из ваших можно подселиться на пару месяцев?",
+		"Сначала четвёртый отдел, потом ревизоры, а теперь ещё и вы? Мне семью кормить надо, хотя вам то какое дело.",
```

```suggestion
		"Сначала четвёртый отдел, потом ревизоры, а теперь ещё и вы? Мне семью кормить надо, хотя ВАМ то какое дело.",
```
Тут необязательное, но в оригинале было вполне уместно.